cmd/jimmsrv/service/service.go

```diff
@@ -451,9 +451,11 @@ func NewService(ctx context.Context, p Params) (*Service, error) {
 		ControllerUUID: p.ControllerUUID,
 		PublicDNSName:  p.PublicDNSName,
 	}
-
 	s.mux.Handle("/api", jujuapi.APIHandler(ctx, &s.jimm, params))
-	RegisterModelHTTPEndpoints(s.mux, &s.jimm)
+	mountHandler(
+		"/model/{uuid}/{type:charms|applications}",
```

Comment:
Nice!

Reply:
What does this syntax do? New to me

Reply:
this handler is mounter for endpoints formatted like:
- /model/{uuid}/charms
-  /model/{uuid}/applications

Nothing very fancy, just the way of doing or in `chi`